.github/release-drafter.yml

```diff
@@ -0,0 +1,45 @@
+name-template: 'v$NEXT_MINOR_VERSION'
+tag-template: 'v$NEXT_MINOR_VERSION'
+categories:
+  - title: '🚀 Features'
+    labels:
+      - "feature"
+      - "enhancement"
+  - title: '💣 Breaking Change'
+    labels:
+      - "breaking"
+      - "change"
+  - title: '🐛 Bug Fixes'
+    labels:
+      - "fix"
+      - "bug"
+      - "bugfix"
+  - title: '📝 Documentation'
+    labels:
+      - "documentation"
+      - "docs"
+  - title: '🔨 Maintenance'
+    labels:
+      - "chore"
+      - "dependencies"
+      - "ci"
+change-template: '- $TITLE @$AUTHOR (#$NUMBER)'
+template: |
+  *Help make the NGINX Ingress Controller better by participating in our [survey](https://forms.office.com/Pages/ResponsePage.aspx?id=L_093Ttq0UCb4L-DJ9gcUM6Dh1A0cORCorfgZAMdkwJUREhJUFAyM1ZHRzZLSzQyMUlCNFhXVkZENy4u)!*
+
+  ## New in NGINX Ingress Controller v$NEXT_MINOR_VERSION
+
+  $CHANGES
+
+  ## Upgrade
```

Comment:
can we follow the existing style for section headers?

Reply:
I was thinking a better separation would help readability